ci/gpu/build.sh

```diff
@@ -49,13 +49,13 @@ conda install "cudatoolkit=$CUDA_REL" \
               "cupy>=6.5.0" "numpy>=1.16" \
               "cudf=${MINOR_VERSION}" "dask-cudf=${MINOR_VERSION}" \
               "dask>=2.8.1" "distributed>=2.8.1" \
+              "ucx" "ucx-proc=*=gpu" \
               -c rapidsai-nightly
 
 # needed for asynccontextmanager in py36
-conda install -c conda-forge "async_generator" "automake" "libtool" \
-                              "cmake" "automake" "autoconf" "cython>=0.29.14,<3.0.0a0" \
-                              "pytest" "pkg-config" "pytest-asyncio" \
-                              "pynvml" "libhwloc" "psutil"
+conda install -c conda-forge "async_generator" "cython>=0.29.14,<3.0.0a0" \
```

Comment:
Does anyone know why we need `async_generator`? AIUI this is a backport package for Python 3.5 (Python 3.6+ already has it). Though the comment above references Python 3.6, we've also already dropped Python 3.6. Can we just drop this too?

Reply:
Trying to drop this in the latest commit.

Reply:
Looks like RAPIDS as a whole doesn't use this either. Proposing we drop this from the integration packages too ( https://github.com/rapidsai/integration/pull/92 ).

Reply:
No failures from dropping it. So looks safe to leave out.

Reply:
Approved the PR for integration